comp/core/internal/params.go

```diff
@@ -48,6 +48,14 @@ type BundleParams struct {
 	// LoggerName is the name that appears in the logfile
 	LoggerName string
 
+	// ExcludeDefaultConfPath determines whether the default config path must be excluded.
```

Comment:
Is this exclusive to DogStatsD? It doesn't seem to be used by the Agent in the component config. I'm not sure it's worth sharing it as a global bundle parameter if I'm right.

Reply:
I have tried to not update the behavior of dogstatsd in this PR but there will be a followup PR to update this code: See https://github.com/DataDog/datadog-agent/pull/13842#discussion_r1001605760.